src/Lumi/Components2/Banner.purs

```diff
@@ -39,6 +40,8 @@ data Banner = Banner
 type BannerProps =
   ( component :: Banner
   , dismissable :: Boolean
+  , icon :: Array JSX
+  , title :: Array JSX
```

Comment:
I believe that using `Maybe JSX` instead of `Array JSX` for these makes more sense. I don't think we want `icon` or `title` to be a collection of things, and we don't predicate over the array itself (e.g. on how the elements should be aligned) where it gets used in `banner`.